libglusterfs/src/logging.c

```diff
@@ -443,7 +443,7 @@ gf_openlog(const char *ident, int option, int facility)
     }
 
     /* TODO: Should check for errors here and return appropriately */
-    setlocale(LC_ALL, "");
+    setlocale(LC_ALL, "en_US.UTF-8");
```

Comment:
We assume this locale is available always? Perhaps we should check for failures and fallback to 'C' if it fails?

Reply:
Done